configure.in

```diff
@@ -520,7 +520,7 @@ int main() {
                 )
             ],
             [
-                AC_MSG_WARN([gRPC not detected, OpenTelemetry module will only support HTTP transport])
+                AC_MSG_WARN([gRPC not detected, OpenTelemetry support will be limited to HTTP transport])
```

Comment:
Since it's no longer the case that one module does both HTTP and gRPC, the original wording no longer made sense. The otlphttp module is always built, so the only thing that is optional is the gRPC portion. To put it another way, we always have OpenTelemetry support in some fashion (HTTP), and support for gRPC transport is conditional upon having the library at or above the minimum version.

Reply:
Maybe `gRPC not detected.  otelgrpc (OpenTelemetry GRPC) module cannot be enabled.` would be clearer then.

Reply:
Most people don't know what a "otlpgrpc" is, so I opted for more general terminology. I think it's pretty clear as it is.

We can discuss in documentation how OpenTelemetry support is implemented, but as this is autoconf, I went with what I felt was succinct and accurate.

Reply:
After discussing with Sproul, this message is more for devs, than operators, so my concerns about the message don't matter.  The wording is fine in this case.